src/main/java/edu/ie3/datamodel/io/source/influxdb/InfluxDbWeatherSource.java

```diff
@@ -30,7 +30,11 @@
   private static final String WHERE = " where ";
   private static final String MEASUREMENT_NAME_WEATHER = "weather";
   private static final int MILLI_TO_NANO_FACTOR = 1000000;
-  private final String coordinateIdColumnName;
+
+  /* Final name of the coordinate id field for use in factories */
+  private final String coordinateIdFieldName;
```

Comment:
why do we track that here as an attribute explicitly. If it's just the value of the `WeatherFactory` we can just access it via the factory

Reply:
That's true